src/db/index.js

```diff
@@ -274,6 +274,34 @@ async function updateFxAData (subscriber, fxaAccessToken, fxaRefreshToken, fxaPr
   return updatedSubscriber
 }
 
+/**
+   * Remove fxa tokens and profile data for subscriber
+   *
+   * @param {object} subscriber knex object in DB
+   * @returns {object} updated subscriber knex object in DB
+   */
+async function removeFxAData (subscriber) {
+  log.debug('removeFxAData', subscriber)
+  const updated = await knex('subscribers')
+    .where('id', '=', subscriber.id)
+    .update({
+      fxa_access_token: null,
+      fxa_refresh_token: null,
+      fxa_profile_json: null
+    })
+    .returning('*')
+  const updatedSubscriber = Array.isArray(updated) ? updated[0] : null
+  if (updatedSubscriber && subscriber.fxa_refresh_token) {
+    log.debug('removeFxAData.destroy-refresh-token', subscriber.fxa_refresh_token)
+    log.debug('removeFxAData.destroy-refresh-token', await destroyOAuthToken({ refresh_token: subscriber.fxa_refresh_token }))
+  }
+  if (updatedSubscriber && subscriber.fxa_access_token) {
+    log.debug('removeFxAData.destroy-access-token', subscriber.fxa_access_token)
+    log.debug('removeFxAData.destroy-access-token', await destroyOAuthToken({ token: subscriber.fxa_access_token }))
```

Comment:
Is all the debug code still needed?  Should we break out the `destroyOAuthToken` functions so that they don't live as part of a debug call?